packages/composerize/src/index.js

```diff
@@ -22,35 +22,62 @@ export default (input: string): ?string => {
     const parsedInput: {
         +_: Array<string>,
         +[flag: string]: RawValue,
-    } = parser(formattedInput);
+    } = parser(formattedInput.replace(/^docker (run|create)/, ''), {
+        configuration: { 'halt-at-non-option': true },
+        boolean: ['i', 't', 'd', 'rm', 'privileged'],
+    });
     const { _: command, ...params } = parsedInput;
 
-    if (command[0] !== 'docker' || (command[1] !== 'run' && command[1] !== 'create')) {
+    if (!formattedInput.startsWith('docker run') && !formattedInput.startsWith('docker create')) {
```

Comment:
fwiw this introduces an edge case that yargs-parser was handling:

```
$ node
> require('yargs-parser')('  docker      create')
{ _: [ 'docker', 'create' ] }
```

I know we already `.trim()` at the edges but if there was extra spacing between `docker` and `run` then we handle that

unlikely to cause issues 99% of the time, but maybe there are cases where folks are hand typing out commands and include an extra space 🤷  and they'd get a confusing error message - hence testing the computed object rather than the input directly, which protects against this

Reply:
oh, right, also should have closed this PR since I did an "all in one" #568  but will recheck in it